# This service is used to upload a new target to the interoperability server.

# Target to add.
Target target

---

# Whether the request was successful.
bool success

# The new target ID.
uint64 id
